paasta_tools/kubernetes_tools.py

```diff
@@ -1584,8 +1584,14 @@ def get_volume_mounts(
         return volume_mounts
 
     def get_boto_secret_name(self) -> str:
-        return _get_secret_name(
-            self.get_namespace(), "boto-key", self.get_service(), self.get_instance()
+        """
+        Namespace is ignored so that there are no bounces with existing boto_keys secrets
+        """
+        # return _get_secret_name(
+        #     self.get_namespace(), "boto-key", self.get_service(), self.get_instance()
+        # )
```

Comment:
same as with `get_boto_new_secret_signature_name`: we can grab this line from this PR/git history rather than leaving commented out code in `master`

Reply:
```suggestion
```